### PR TITLE
Migrate cassandra test from acceptance to dockertest

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

File: builtin/logical/cassandra/backend_test.go [hunk 1]
```diff
@@ -4,37 +4,115 @@ import (
...
 	"github.com/mitchellh/mapstructure"
+	dockertest "gopkg.in/ory-am/dockertest.v2"
 )
 
+var (
+	testImagePull sync.Once
+)
+
+func prepareTestContainer(t *testing.T, s logical.Storage, b logical.Backend) (cid dockertest.ContainerID, retURL string) {
+	if os.Getenv("PG_URL") != "" {
```
Comment (jefferai) [inline builtin/logical/cassandra/backend_test.go]: Probably this should be CASSANDRA_URL or whatever Cassandra uses.

**Comment by jefferai** [comment]
This part looks good, but the next step is to remove the acceptance `true` bool from the various test cases/steps and see what breaks.

**Comment by briankassouf** [comment]
@jefferai Fixed the env variable. However I do believe all the tests are running and passing now. I removed all the `AcceptanceTest: true` lines.

**Comment by jefferai** [comment]
Yep, they're all passing!

**Pull request merged**